### ISSUE TITLE
CODE RED Esports Computers

### ISSUE THREAD

**Issue by CodingPenguin1** [wrightedu/Programmers-Guide-to-the-Galaxy#83]
Front 5 bsod
Full overhaul

**Comment by CodingPenguin1** [comment]
@CodingPenguin1 knows

**Comment by mbozada** [comment]
Two systems ran into the deepfreeze windows update issue. Look to Issue #82 for possible solutions to that.

Other than that, the problem most likely has to do with BattlEye AntiCheat based off of crash reports. We don't know how to recreate what led to BSOD or how to fix the problem. It could be resolved with the above when we have a new version of deepfreeze set up.

Finally, there were reported issues of systems freezing without full on crashing. @CodingPenguin1 was unaware of them, but @tux2603 seemed to know what was going on. That may need to be a separate issue if the cause doesn't also have to do with BattlEye. Or, if the solution is a reimaging of the systems in the clubroom, then both this issue as well as issue #82 could be merged into a single issue.

Feel free to edit this or add to the issue for anything I missed or described incorrectly.

**Comment by mbozada** [comment]
Moving to backlog while waiting on Owen to be more available in October and Deepfreeze update.
